cypress/integration/MultiSelect/clearing/index.js

```diff
@@ -0,0 +1,18 @@
+import '../common'
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a clearable MultiSelect with a selection is rendered', () => {
+    cy.visitStory('MultiSelect', 'With clear button and selection')
+    cy.window().then(win => (win.onChange = cy.stub()))
```

Comment:
This is already set in the `Given an onChange handler is attached` step

Reply:
Ah yeah good catch, I missed that one